deploy/core/main.js

```diff
@@ -152,7 +152,7 @@ if (process.env["IPC_DEBUG"]) {
       oldSend.apply(window.webContents, arguments);
     };
   };
-  let oldCreateWindow = createWindow;
+  let oldCreateWindow = createWindow();
```

Comment:
The `BrowserWindow` constructor returns an instance not a function so you don't to add the parentheses.

Reply:
Yes, this is a mistake. This was unintentional.